### PR TITLE
Windows Wheels

### PR DESCRIPTION
**Pull request by nabobalis**
In New York
Concrete jungle where dreams are made of
There's nothing you can't do

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -9,6 +9,23 @@ requires = [
...
+test-command = 'pytest --pyargs pyflct'
+
+[tool.cibuildwheel.linux]
+before-all = 'yum install -y fftw-devel; git clean -xfd -e wheelhouse'
+
+[tool.cibuildwheel.macos]
+before-all = 'brew install fftw; git clean -xfd -e wheelhouse'
+
+[tool.cibuildwheel.windows]
+before-all = '%CONDA%\Scripts\conda.exe install fftw && git clean -xfd -e wheelhouse'
```
Comment (Cadair) [inline pyproject.toml:25]: oh goodness. Is this binary compatible with manylinux2014 or whatever we are building against?

Comment (nabobalis) [reply pyproject.toml:25]: Who knows, I don't that's for sure.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -69,7 +72,7 @@ stages:
         envs:
           - macos: py38
           - linux: py39
-          - linux: py310-conda
+          - windows: py310-conda
```
Comment (nabobalis) [inline azure-pipelines.yml:75]:
```suggestion
          # - windows: py310-conda
```

File: azure-pipelines.yml [hunk 3]
```diff
@@ -98,4 +101,8 @@ stages:
             - wheels_cp39-macosx*
             - wheels_cp310-macosx*
             - wheels_cp311-macosx*
+            - wheels_cp38-windows*
+            - wheels_cp39-windows*
+            - wheels_cp310-windows*
+            - wheels_cp311-windows*
```
Comment (nabobalis) [inline azure-pipelines.yml:107]:
```suggestion
            - wheels_cp38-win*
            - wheels_cp39-win*
            - wheels_cp310-win*
            - wheels_cp311-win* 
```

File: azure-pipelines.yml [hunk 4]
```diff
@@ -1,7 +1,10 @@
 name: $(BuildDefinitionName)_$(Date:yyyyMMdd)$(Rev:.rr)
 variables:
-  CIBW_BEFORE_BUILD_LINUX: "yum -y install fftw-devel; git clean -xfd -e wheelhouse"
-  CIBW_BEFORE_BUILD_MACOS: "brew install fftw; git clean -xfd -e wheelhouse"
+  CIBW_BEFORE_BUILD_LINUX: "yum -y install fftw-devel && git clean -xfd -e wheelhouse"
+  CIBW_BEFORE_BUILD_MACOS: "brew install fftw && git clean -xfd -e wheelhouse"
+  CIBW_BEFORE_ALL_WINDOWS : '%CONDA%\Scripts\conda.exe install fftw'
+  CIBW_BEFORE_BUILD_WINDOWS: "pip install delvewheel"
+  CIBW_REPAIR_WHEEL_COMMAND_WINDOWS: "delvewheel repair --add-path %CONDA%\Library\bin -w {dest_dir} {wheel}"
```
Comment (nabobalis) [inline azure-pipelines.yml:8]:
```suggestion
  CIBW_SKIP: "*-win32 *-manylinux_i686"

```

**Comment by nabobalis** [comment]
Now to debug if this is even possible.

File: pyproject.toml [hunk 5]
```diff
@@ -17,6 +17,11 @@ before-all = "yum -y install fftw-devel; git clean -xfd -e dist"
 [tool.cibuildwheel.macos]
 before-all = "brew install fftw; git clean -xfd -e dist"
 
+[tool.cibuildwheel.windows]
+before-all = "%CONDA%\\Scripts\\conda.exe install fftw; git clean -xfd -e dist"
```
Comment (nabobalis) [inline pyproject.toml:21]:
```suggestion
before-all = "conda.exe install fftw; git clean -xfd -e dist"
```

**Comment by nabobalis** [comment]
On Azure the plan was to use the conda install I had access to to install the binary FFTW files and add them to the wheel and see if they actually worked.

On actions, I don't think that is possible. So I am using the pyfftw code to see if I can use it for windows here.

**Pull request closed without merging**